js/config/styles 2.js

```diff
@@ -0,0 +1,48 @@
+export const colors = {
+  BACKGROUND: "#ffffff",
```

Comment:
you can make a multiline comment saying what are the main colors with their respective hash values.